pkg/reconciler/autoscaling/reconciler.go

```diff
@@ -89,7 +90,7 @@ func (c *Base) ReconcileSKS(ctx context.Context, pa *pav1alpha1.PodAutoscaler, d
 		if !equality.Semantic.DeepEqual(tmpl.Spec, sks.Spec) {
 			want := sks.DeepCopy()
 			want.Spec = tmpl.Spec
-			logger.Info("SKS changed; reconciling:", sksName)
+			logger.Infof("SKS %s changed; reconciling, diff: %s", sksName, cmp.Diff(want.Spec, tmpl.Spec))
```

Comment:
Use `kmp` in the controllers. `cmp` has the potential to panic.

Either SafeDiff or ShortDiff should work here.

https://github.com/knative/pkg/blob/master/kmp/diff.go#L41

Reply:
I disagree that it's unsafe here but ok :)